addScript.js

```diff
@@ -3,8 +3,15 @@
 	Author Tobias Koppers @sokra
 */
 module.exports = function(src) {
-	if (typeof execScript !== "undefined")
-		execScript(src);
-	else
-		eval.call(null, src);
+	try {
+		if (typeof eval !== "undefined") {
+			eval.call(null, src);
+		} else if (typeof execScript !== "undefined") {
+			execScript(src);
+		} else {
+			console.log("no eval function available");
```

Comment:
@michael-ciniawsky it can be break execution there `eval` and `execScript` not allowed (CSP). Maybe best output `error` than break execution?